backend/de.metas.salescandidate.base/src/main/sql/postgresql/system/45-de.metas.salescandidate/5635380_sys_gh12840_C_OLCand_add_AD_Issue_ID_col.sql

```diff
@@ -0,0 +1,35 @@
+;
+
+-- 2022-04-14T09:46:42.512Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Table_Process (AD_Client_ID,AD_Org_ID,AD_Process_ID,AD_Table_ID,AD_Table_Process_ID,Created,CreatedBy,EntityType,IsActive,Updated,UpdatedBy,WEBUI_DocumentAction,WEBUI_IncludedTabTopAction,WEBUI_ViewAction,WEBUI_ViewQuickAction,WEBUI_ViewQuickAction_Default) VALUES (0,0,540173,540244,541104,TO_TIMESTAMP('2022-04-14 12:46:42','YYYY-MM-DD HH24:MI:SS'),100,'de.metas.ordercandidate','Y',TO_TIMESTAMP('2022-04-14 12:46:42','YYYY-MM-DD HH24:MI:SS'),100,'Y','N','Y','N','N')
+;
+
+-- 2022-04-14T14:10:27.203Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Column (AD_Client_ID,AD_Column_ID,AD_Element_ID,AD_Org_ID,AD_Reference_ID,AD_Table_ID,ColumnName,Created,CreatedBy,DDL_NoForeignKey,Description,EntityType,FacetFilterSeqNo,FieldLength,Help,IsActive,IsAdvancedText,IsAllowLogging,IsAlwaysUpdateable,IsAutoApplyValidationRule,IsAutocomplete,IsCalculated,IsDimension,IsDLMPartitionBoundary,IsEncrypted,IsExcludeFromZoomTargets,IsFacetFilter,IsForceIncludeInGeneratedModel,IsGenericZoomKeyColumn,IsGenericZoomOrigin,IsIdentifier,IsKey,IsLazyLoading,IsMandatory,IsParent,IsSelectionColumn,IsShowFilterIncrementButtons,IsShowFilterInline,IsStaleable,IsSyncDatabase,IsTranslated,IsUpdateable,IsUseDocSequence,MaxFacetsToFetch,Name,SelectionColumnSeqNo,SeqNo,Updated,UpdatedBy,Version) VALUES (0,582800,2887,0,19,540244,'AD_Issue_ID',TO_TIMESTAMP('2022-04-14 17:10:26','YYYY-MM-DD HH24:MI:SS'),100,'N','','de.metas.ordercandidate',0,10,'','Y','N','Y','N','N','N','N','N','N','N','N','N','N','N','N','N','N','N','N','N','N','N','N','N','N','N','Y','N',0,'Probleme',0,0,TO_TIMESTAMP('2022-04-14 17:10:26','YYYY-MM-DD HH24:MI:SS'),100,0)
+;
+
+-- 2022-04-14T14:10:27.214Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Column_Trl (AD_Language,AD_Column_ID, Name, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy,IsActive) SELECT l.AD_Language, t.AD_Column_ID, t.Name, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy,'Y' FROM AD_Language l, AD_Column t WHERE l.IsActive='Y'AND (l.IsSystemLanguage='Y') AND t.AD_Column_ID=582800 AND NOT EXISTS (SELECT 1 FROM AD_Column_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Column_ID=t.AD_Column_ID)
+;
+
+-- 2022-04-14T14:10:27.265Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+/* DDL */  select update_Column_Translation_From_AD_Element(2887) 
+;
+
+-- 2022-04-14T14:10:30.748Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+/* DDL */ SELECT public.db_alter_table('C_OLCand','ALTER TABLE public.C_OLCand ADD COLUMN AD_Issue_ID NUMERIC(10)')
+;
+
+-- 2022-04-14T14:10:31.069Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+ALTER TABLE C_OLCand ADD CONSTRAINT ADIssue_COLCand FOREIGN KEY (AD_Issue_ID) REFERENCES public.AD_Issue DEFERRABLE INITIALLY DEFERRED
```

Comment:
generally don't have FK-constraints to AD_Issue (there is also a column in AD_column, pls untick it).
we want to be able to trunctate AD_Issue, e.g. for "minidumps"

Reply:
right, I removed the FK from here